### DISCUSSION TITLE
Daily 19/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#99] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve realizando el diagrama de casos de uso general, corrigiendo documentación, agregando las mismas a la wiki y subiendo documentación y diagramas al repositorio. No me dio el tiempo para agregar las tareas a las issue, así que en eso voy a andar hoy. También quiero continuar con Angular y el formulario para presentar el proyecto.

**Comment by rpgrca** [comment]
Armé una clase para testear artículo (también tomé esa tarea del back-end para usarla de ejemplo en un video). Tal vez si tenemos que usar el ORM de Django podemos usar las mismas clases o habrá que copiar los atributos en otra clase, todavía no me fijé. Tendríamos que determinar el texto del README así que habría que definir el texto de la presentación del proyecto de la heladería y el porqué, además de subir los gráficos en la Wiki que falten y borrar los que no vamos a usar, voy a dar una mano con eso hoy.

**Comment by BraianTroncoso** [comment]
Ayer me puse a completar un par de clases en back del proyecto que tengo que confirmar que esten bien antes de pushear.
Hoy voy a ver unas clases de Java de la UTN y si puedo veo algo de Angular(Cosa que no creo).
